Manual.md

```diff
@@ -90,20 +45,22 @@ The basic components and data flow for a single container looks like this. Some
 
 ```
 
+In the filesystem a container is a single directory passed via $PGARCHIVE in the environment to all `pgarchive` invocations. Pathes in the diagram above are assumed to be rooted at this directory.
```

Comment:
i think it needs to be better described what a _container_ is. you're introducing the name of the concept in the first section above but the description of what makes up a container is scattered across multiple paragraphs. this is really not helping if you're reading it this manual for the first time. also in the architecture diagram you don't find "the container". 

i'd make it clear from the beginning of what a container consists of and also add the corresponding "bubble" in a diagram for a less steep learning curve.